adders/prettier/config/tests.ts

```diff
@@ -0,0 +1,14 @@
+import { defineAdderTests } from "@svelte-add/core";
+import { options } from "./options";
+
+export const tests = defineAdderTests({
+    files: [],
+    options,
+    optionValues: [],
+    tests: [
+        {
+            name: "todo",
+            run: async () => {},
+        },
```

Comment:
I don't think we will be able to test this at all. The only think that could work here is using the postconditions. 

IIRC tests for an adder should fail, if a precondition fails

Reply:
I was thinking something along the lines of reading the config files and confirming they were updated/created

Reply:
Yeah, that could work

Reply:
hrmm, I'm running into a bit of a wall right now where I can't really read the contents of the `.prettierrc` because the `run` hook in tests doesn't provide a `cwd`. Perhaps we can save this for later until #461 is implemented?